index.html

```diff
@@ -46,17 +46,18 @@
 					</ul>
 					<div class="languagepicker roundborders large">
 
-						<a class="lang waves-effect waves-light btn blue-grey darken-2" data-href="language">Select your language</a>
-							<a class="translate translate-language-choice waves-effect waves-light btn blue-grey darken-2" id="en">
+						<div class="select-your-language"><a class="lang waves-effect waves-light btn blue-grey darken-2" data-href="language">Select your language</a></div>
```

Comment:
If you remove the div and add the class to a like `<a class="select-your-language lang waves-effect waves-light btn blue-grey darken-2">` Does it still work?

Reply:
Nope then it does not cover the entire area only the written text since <a> is inline .